steam-wrapper

```diff
@@ -1,15 +1,11 @@
 #!/bin/sh
 
 _STEAM_HOME="$HOME/.var/app/com.valvesoftware.Steam/home"
-
-if [ ! -d "$_STEAM_HOME" ]; then
-    # Create a custom HOME so that games can write wherever they want and it persists
-    mkdir -p $_STEAM_HOME
-    cd $_STEAM_HOME
-    ln -s ../../../../.local .
-    ln -s ../../../../.steam .
-    ln -s ../../../../.var .
+if [ -d $_STEAM_HOME ]; then
+    export GLOBIGNORE=".:.."
```

Comment:
I think this is a bashism, POSIX shell requires you to first set variable, then export the setted variable. Hashbang declares POSIX shell so this might in the future break